src/ResourceManager/LocationBasedServices/Commands.LocationBasedServices/help/Get-AzureRmLocationBasedServicesAccountKey.md

```diff
@@ -0,0 +1,126 @@
+---
+external help file: Microsoft.Azure.Commands.LocationBasedServices.dll-Help.xml
+Module Name: AzureRM.LocationBasedServices
+online version:
+schema: 2.0.0
+---
+
+# Get-AzureRmLocationBasedServicesAccountKey
+
+## SYNOPSIS
+Gets the API keys for an account. These keys are the authentication mechanism used in subsequent calls to Azure Location Based Services.
+
+## SYNTAX
+
+### NameParameterSet
+```
+Get-AzureRmLocationBasedServicesAccountKey [-ResourceGroupName] <String> [-Name] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### InputObjectParameterSet
+```
+Get-AzureRmLocationBasedServicesAccountKey [-InputObject <PSLocationBasedServicesAccount>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzureRmLocationBasedServicesAccountKey** cmdlet gets the API keys for a provisioned Location Based Services account.
+
+A Location Based Services account has two API keys: Primary and Secondary.
+The keys enable interaction with the Location Based Services account endpoint.
+
+Use [New-AzureRmLocationBasedServicesAccountKey](New-AzureRmLocationBasedServicesAccountKey.md) to regenerate a key.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Get-AzureRmLocationBasedServicesAccountKey -ResourceGroupName MyResourceGroup -Name MyAccount
+
+PrimaryKey                                  SecondaryKey
+----------                                  ------------
+yKp9encJJQyJMlxgj0gc4n5Iz9DheQyxt020Rwnd3a8 AeE4SoxPk6BrYZgRlqA8uuchLXzkTEHTpm3_iMaml2g
```

Comment:
You might want to ensure that these are obviously redeacted (xxxxxxxx, yyyyyyyyyy, ***********, etc.)

Reply:
ya wasn't sure what best practice was.   These keys are just examples; they won't work.
but i can redact.